modules/bitgo/src/wallet.ts

```diff
@@ -449,9 +449,10 @@ Wallet.prototype.address = function (params, callback) {
 
   const url = this.url('/addresses/' + params.address);
 
-  return this.bitgo.get(url)
-    .result()
-    .nodeify(callback);
+  return Bluebird.resolve(
```

Comment:
What is the needed response that is being returned that is currently being returned with the previous code?

Reply:
previously it would throw an error saying `this.bitgo.get(...).result(...).nodeify is not a function`. Based on other places where `.nodify()` was used, it seemed like something specific to Bluebird requests.